easybuild/easyconfigs/n/NVHPC/NVHPC-21.3.eb

```diff
@@ -0,0 +1,76 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'NVHPC'
+version = '21.3'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the HPC SDK Software License Agreement (https://docs.nvidia.com/hpc-sdk/eula/index.html)
+accept_eula = True
```

Comment:
I don't think we should set this ever in central easyconfigs (and we should remove it from the NVHPC easyconfigs that do set it).

You can configure EasyBuild to accept EULAs blindly if you want to via `--accept-EULA=*`, but we shouldn't do this on behalf of the user by default...

Reply:
I need to change this too, sorry.

Reply:
I have noticed where this comes from. Basically, Alex Domingo added it with his pull request #11920 and I only inherited it while doing 21.2 and 21.3.